README.md

```diff
@@ -30,7 +30,7 @@ To get started you will first need to satisfy the following pre-requisites:
 ## Pre-requisites
 1. An OpenShift cluster that is 4.9 or higher.
 2. You will need to be logged into this cluster as [cluster admin] via [oc client].
-3. The OpenShift Cluster must have OpenShift Pipelines 1.7.2 or higher installed. Instructions [here][OCP Pipelines Operator].
+3. The OpenShift Cluster must have OpenShift Pipelines 1.8 or higher installed. Instructions [here][OCP Pipelines Operator].
```

Comment:
The OpenShift Cluster must have OpenShift Pipelines 1.8 or higher installed. Use the `pipelines-1.8 `channel.

Reply:
Alternative: We always recommend version 1.8.x for all OCP 4 minor versions